sonar-php-plugin/src/test/java/org/sonar/plugins/php/PHPSensorTest.java

```diff
@@ -432,7 +432,20 @@ public void test_issues() {
   }
 
   @Test
-  public void test_regex_issues() {
+  public void testSuppressWarningsIssues() {
+    final String FILE_TO_ANALYZE = "suppressWarnings.php";
```

Comment:
I think we only use SCREAM_CASE for final static variables

Reply:
Indeed! I was thinking it was related to final alone.

Reply:
Or not, not totally sure what happened here XD There were two tests before related to this file `suppressWarnings.php`, now that there is only one, this variable does not make sense anymore, I will remove it and use the literal string directly.